src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: (1, 200px);
+
+  gap: 30px 48px;
+  font-family: $font-family-roboto;
+
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    width: 200px;
+    border: 1px solid #f3f3f3;
+    transition-duration: 300ms;
+
+    &_image {
+      display: block;
+      margin: 0 auto 40px;
+      width: 160px;
+      height: 134px;
+    }
+
+    &_name {
+      display: block;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      width: 166px;
+      color: $main-color;
+      margin-bottom: 4px;
+      margin-top: 0;
+    }
+    &_product-code {
+      margin-bottom: 16px;
+      margin-top: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $secondary-color;
+    }
+
+    &_review-count {
+      text-align: right;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $main-color;
+    }
+    &_review-container {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+    }
+
+    &_price {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 16px;
+      height: 18px;
+
+      &-text {
+        font-size: 12px;
+        font-weight: 400;
+        line-height: 18px;
+        color: $secondary-color;
+      }
+
+      &-value {
+        font-size: 16px;
+        font-weight: 700;
+        line-height: 18px;
+        color: $main-color;
+      }
+    }
+    &_buy-link {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-bottom: 16px;
+      box-sizing: border-box;
+      height: 40px;
+      border: 1px solid $blue-color;
+      border-radius: 5px;
+
+      font-size: 14px;
+      line-height: 16px;
+      color: $white-color;
+      font-weight: 700;
+      text-decoration: none;
+      text-transform: uppercase;
+
+      background-color: $blue-color;
+      transition-duration: 300ms;
+    }
+    &_buy-link:hover {
+      background-color: $white-color;
+      color: $blue-color;
+    }
+    &:hover {
+      transform: scale(1.2);
+    }
```

Comment:
The transform scale on hover for the entire card might not be the best user experience, as it can cause the layout to shift. If a scaling effect is desired, consider applying it to a visual element inside the card, such as an image, rather than the entire card.